contracts/modules/licensing/PILicenseTemplate.sol

```diff
@@ -0,0 +1,472 @@
+// SPDX-License-Identifier: BUSL-1.1
+
+pragma solidity 0.8.23;
+
+// external
+import { Strings } from "@openzeppelin/contracts/utils/Strings.sol";
+import { IERC165 } from "@openzeppelin/contracts/utils/introspection/IERC165.sol";
+import { ERC165Checker } from "@openzeppelin/contracts/utils/introspection/ERC165Checker.sol";
+import { ReentrancyGuardUpgradeable } from "@openzeppelin/contracts-upgradeable/utils/ReentrancyGuardUpgradeable.sol";
+
+// contracts
+import { IHookModule } from "../../interfaces/modules/base/IHookModule.sol";
+import { ILicenseRegistry } from "../../interfaces/registries/ILicenseRegistry.sol";
+import { IRoyaltyModule } from "contracts/interfaces/modules/royalty/IRoyaltyModule.sol";
```

Comment:
Use relative path, otherwise importing core repo from an external repo can cause foundry remapping issues.

Reply:
changed to relative.